src/features/darkMode.tsx

```diff
@@ -0,0 +1,25 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+type DarkMode = {
+  darkMode: boolean,
+};
+
+const initialState: DarkMode = {
+  darkMode: false,
+};
+
+const darkModeSlice = createSlice({
+  name: 'darkMode',
+  initialState,
+  reducers: {
+    setDarkMode: (state, action: PayloadAction<boolean>) => {
+      return {
+        ...state,
+        darkMode: action.payload,
+      };
+    },
+  },
+});
+
+export default darkModeSlice.reducer;
```

Comment:
It`s a good approach to implement darkMode reducer in a separate file, makes the code more modular and easy to maintain!